<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanVacationView</class>
 <widget class="QWidget" name="PlanVacationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#chooseType {
background-color: #97FFB0;
}

QWidget#planVacation
{
background-color: #97FFB0;
}

QWidget#vacationDistance
{
background-color: #97FFB0;
}

QWidget#restaurantNumber_label
 {
color:black;
font: italic;
}

QWidget#rightBar_3
{
background-color: #303030;
}

QWidget#planTripView
{
 background-color: #99C2E8; 
}

QWidget#teamHeader
{
background-color: #303030;
color:white;
}

QWidget#background {
background-color: white;
border: 1px solid black;
}

QLabel#distanceLabel {
font-size: 24px;
font: bold;
color:black;
}

QLabel#distance {
font-size: 24px;
color:black;
}</string>
  </property>
  <widget class="QStackedWidget" name="planVacationStack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>710</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="chooseType">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QWidget" name="planVacation">
    <widget class="QLabel" name="teamHeader">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>520</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft Sans Serif</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Plan Vacation View</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tripMap"/>
   <widget class="QWidget" name="activeVacation"/>
   <widget class="QWidget" name="vacationReceipt"/>
   <widget class="QWidget" name="vacationDistance">
    <widget class="QWidget" name="rightBar_3" native="true">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>90</width>
       <height>600</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
    <widget class="QWidget" name="background" native="true">
     <property name="geometry">
      <rect>
       <x>49</x>
       <y>140</y>
       <width>521</width>
       <height>161</height>
      </rect>
     </property>
     <widget class="QLabel" name="distance">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>miles</string>
      </property>
     </widget>
     <widget class="QLabel" name="distanceLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Total Distance:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="ContinueToNext">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>410</y>
       <width>115</width>
       <height>26</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:black;
background:#D5D5D5;
border-radius:5px;</string>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
